Microsoft.Toolkit.Uwp.SampleApp/Pages/About.xaml

```diff
@@ -186,6 +186,9 @@
                                     <HyperlinkButton Style="{StaticResource AboutHyperlinkButtonStyle}" NavigateUri="https://unsplash.com/photos/105m46GatAg" Click="Link_Clicked">
                                         <TextBlock>Background photo by @apertism</TextBlock>
                                     </HyperlinkButton>
+                                    <HyperlinkButton Style="{StaticResource AboutHyperlinkButtonStyle}" NavigateUri="https://www.lottiefiles.com/47-gears" Click="Link_Clicked">
+                                        <TextBlock>Lottie Files</TextBlock>
```

Comment:
Sorry to be nitpicky, but per their [license suggestion](https://www.lottiefiles.com/license) maybe use `Gears example / LottieFiles` as the text?